extra-libs/dotnet-host/autobuild/build

```diff
@@ -1,9 +1,9 @@
-abinfo "Deploy files"
-mkdir -p "$PKGDIR"/usr/bin
-mkdir -p "$PKGDIR"/usr/lib/dotnet
-mkdir -p "$PKGDIR"/usr/share/doc/dotnet
-cp "$SRCDIR"/dotnet "$PKGDIR"/usr/lib/dotnet
-cp "$SRCDIR"/LICENSE.txt "$PKGDIR"/usr/share/doc/dotnet
-cp "$SRCDIR"/ThirdPartyNotices.txt "$PKGDIR"/usr/share/doc/dotnet
+abinfo "Deploying files ..."
+mkdir -pv "$PKGDIR"/usr/bin
+mkdir -pv "$PKGDIR"/usr/lib/dotnet
+mkdir -pv "$PKGDIR"/usr/share/doc/dotnet
+install -Dvm755 "$SRCDIR"/dotnet "$PKGDIR"/usr/lib/dotnet
+install -Dvm644 "$SRCDIR"/LICENSE.txt "$PKGDIR"/usr/share/doc/dotnet
+install -Dvm644 "$SRCDIR"/ThirdPartyNotices.txt "$PKGDIR"/usr/share/doc/dotnet
 # cp dotnet.1.gz "$PKGDIR"/usr/share/man/man1
-ln -s /usr/lib/dotnet/dotnet "$PKGDIR"/usr/bin/dotnet
+ln -sv /usr/lib/dotnet/dotnet "$PKGDIR"/usr/bin/dotnet
```

Comment:
It's probably more reasonable to use a relative symlink target here, i.e. `../lib/dotnet/dotnet`.